tests/test_precompiled.py

```diff
@@ -762,10 +790,10 @@ def test_precompiled_sanitise_pdf_without_notify_tag(client, auth_header):
             **auth_header
         }
     )
-
     assert response.status_code == 200
+    json_data = json.loads(response.get_data())
```

Comment:
can we add checks to this test `test_precompiled_sanitise_pdf_without_notify_tag` and `test_precompiled_validation_endpoint_blank_pdf` (as the two simplest happy path tests i think) that just assert the entire json layout? just so we can be sure that the errors dict is empty, page count is populated, etc?

Reply:
I added the check to `test_precompiled_sanitise_pdf_without_notify_tag`, but not to the other test since the other test uses validate endpoint that will become obsolete when we ship single file upload through the interface, as we talked on Thursday. I hope this is ok 👍